### ISSUE TITLE
Deprecated field warning incorrectly showing up

### ISSUE THREAD

**Issue by halvespereira** [graphql/graphiql#1760]
This warning for deprecated field should not be showing up as this field is not deprecated. Is there a way to disable this warning? 

![image](https://user-images.githubusercontent.com/62767337/104196414-5c8a8780-53e9-11eb-8aa9-cc6ffa8f08ca.png)

**Comment by acao** [comment]
hmmm thats unusual... do you have any other deprecated fields in your schema?

**Comment by halvespereira** [comment]
We just searched for it and there aren't any deprecated fields in this schema. I've tried uninstalling and installing an older version 0.17.5 for example and it's still there

**Comment by halvespereira** [comment]
Any update on this? Is there a way to ignore this feature? Thanks

**Comment by acao** [comment]
unfortunately I don't have enough information to reproduce the bug. deprecated fields are working fine for us in all implementations I've looked at. if you can provide a repository where I can see this bug occurring, then I can help you!

it may just be your schema introspection response has an issue, where it's accidentally marking your Type revolvers as deprecated?

**Comment by etiennedi** [comment]
*tl;dr; If anyone else runs into this issue while using `graphql-go`, make sure to update to version `v0.7.9` as a workaround. This fixed it for us.*

---

Hi @acao thanks for your help so far.

To add some more context to what @halvespereira has already been saying:

We have since been able to find a work-around. We noticed that while `isDeprecated` was always set to `false`, the `deprecationMessage` was set to an empty string, as opposed to explicit `null`. 

Please also see [this commit message for details](https://github.com/semi-technologies/weaviate/commit/6004dcc4df57e5cf5143d210dc875aec0eda2fef) which tries to explain the whole history of changes and how we ran into this issue.

Could it be that - [as suggested in here](https://github.com/graphql-go/graphql/issues/504#issuecomment-549080559) - graphiql somehow depends on an old version of `graphql-js` where this bug has not yet been fixed?

**Comment by acao** [comment]
@etiennedi thanks for all the info, and good question! thats up to the implementation - graphiql does not ship with a specific version of `graphql`, it lists it as a `peerDependency`, and specifies a range. so it must be that these implementations are importing an outdated version of `graphql-js` library? 

If you're able to track down the particular version in which this bug occurs and the first patch version, I can set the `peerDependencies` range to warn developers if they use any version before the fix for this at least?

**Comment by acao** [comment]
https://graphiql-test.netlify.com < this is our latest release, FYI!

**Comment by acao** [comment]
@halvespereira were you able to resolve your issue, or were you able to re-create the issue with the latest version?

it may be that some other tooling is forcing this particular graphql tooling that is causing issues with your build?

**Comment by acao** [comment]
@etiennedi looking at the links you provided, I think the issue is best resolved via `graphql-go`'s graphiql implementation, if this is still an issue. again, because we don't have a fixed dependency on any version of `graphql-js`, I cannot help you. let me know if there are any questions the `graphql-go` folks have though!
